docs/product-installation-guide/src/main/asciidoc/clustering-con.adoc

```diff
@@ -1,15 +1,64 @@
 [id='clustering-con']
-= Clustering with {PRODUCT} for design-time development environments
+= Clustering  {PRODUCT} 
+
+By clustering two or more computers, you have the benefits of high availability, enhanced collaboration, and load balancing. High availability decreases the chance of a loss of data when a single computer fails because data on all of the computers (nodes) of the cluster is replicated on each node of the cluster. When a computer fails, another computer fills the gap by providing a copy of the data that was on the failed computer. When the failed computer comes online again, it resumes its place in the cluster. Load balancing shares the computing load across the nodes of the cluster. Doing this improves the overall performance of the cluster.
+
+There are several ways that you can cluster {PRODUCT} components. This document describes how to cluster the following scenarios:
+
+* Clustering a {PRODUCT} development environment
+* Clustering a {KIE_SERVER} standalone runtime environment
+* Clustering a {KIE_SERVER} runtime environment that is managed by the standalone {KIE_SERVER} controller
+
+== Clustering {PRODUCT} in a development environment
```

Comment:
not 'in a development environment', the correct word would be 'for a development environment'